cnc_ctrl_v1/Motion.cpp

```diff
@@ -263,7 +263,7 @@ int   arc(const float& X1, const float& Y1, const float& X2, const float& Y2, co
             theta -= 2*pi;
         }
     }
-    if ((sign(theta) != sign(direction)) || ((abs(chordHeight) < .01) && (abs(theta) < 0.5))) {
+    if ((sign(theta) != sign(direction)) || ((abs(chordHeight) < .01) && (abs(theta) < 0.5)) || (radius > 25400)) {
```

Comment:
How is the number 25400 computed? Not asking because I think there is anything wrong with it, just curious

Reply:
That’s 1000 inched turned into millimeters 😀

Reply:
Got it 😀 Thanks